app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -163,22 +162,8 @@ AcuantContext.displayName = 'AcuantContext';
 /**
  * Returns a found AcuantJavascriptWebSdk
  * object, if one is available.
- * Depending on the SDK version,
- * will use either startWorkers (11.8.2) or start (11.9.1)
  */
 const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
-  if (
-    window.AcuantJavascriptWebSdk &&
-    typeof window.AcuantJavascriptWebSdk.startWorkers === 'function' &&
-    typeof window.AcuantJavascriptWebSdk.start !== 'function'
-  ) {
-    return {
-      ...window.AcuantJavascriptWebSdk,
-      start(...args) {
-        window.AcuantJavascriptWebSdk.startWorkers?.(...args);
-      },
-    };
-  }
   if (window.AcuantJavascriptWebSdk && typeof window.AcuantJavascriptWebSdk.start === 'function') {
```

Comment:
We are no longer supporting older versions of Acuant that don't include start as a function.  I included this `typeof window.AcuantJavascriptWebSdk.start === 'function'` check just in case, but I am curious to hear what other people think about it, too.

Reply:
Seems unnecessary now, I'd vote to remove it

Reply:
@zachmargolis I agree.  I removed the check.